pyinterprod/interpro/contrib/gp.py

```diff
@@ -0,0 +1,40 @@
+import cx_Oracle
+
+
+def import_gp(cur: cx_Oracle.Cursor, file_path: str):
+    gp = []
+    with open(file_path, "rt") as fh:
+        for line in fh:
+            if line.startswith("AC"):
+                entry_ac = line.split()[1]
+            elif line.startswith("DE"):
+                description = line.split(" ", maxsplit=1)[1].strip()
+            elif line.startswith("EV"):
+                ev = line.split()[1].split(";")[0]
+                if ev.startswith("IPR"):
+                    gp.append((ev, "h", description, entry_ac))
```

Comment:
I would use a regular expression:

```suggestion
                _, evidence = line.split(maxsplit=1)
                m = re.match(r"IPR\d+", evidence)
                if m:
                    gp.append((m.group(0), "h", description, entry_ac)(
```